qmd/tables.qmd

```diff
@@ -1,9 +1,205 @@
+<!--Source constants file until I get clarity for environment usage across entire package or if sourcing is enough -->
+
+`r source("../R/package_constants.R")`
+
+<!--` reference1 -->
+
 # Tables {.unnumbered}
 
-The list of tables in the three main groups (partType = tables, class = results, prgramDesc, lookup).
+The list of tables below are for ODM version `r pkg.env$dictionary_version_used`.
+
+There are 21 tables in the ODM.<!--# I've added explanatory text. This text should be moved to our explanatory documents, but I've placed here as a temporary solution until we generate those documents. -->
+
+Users can customize the tables for different purposes. The format of the tables in this reference document is for the full relational database model. The comprehensive list of tables includes all mandatory and optional headers. The table format is used for drata storage and they are informally described as "long" tables with one measure, method or attribute for each row.
+
+Users can modify or create their own table format for data entry, analyses, or reporting. Commonly, these tables are a "wide" format where there is one day per row and multiple measures or attributes per row. How to generate wide tables is not included in this section.
+
+There are three types of tables:
+
+1\) **Regular report tables**. The `measues` and `samples` tables are used day-to-day reporting of new measurements and information about how and when samples are collected.
+
+Three tables support regular reporting. `measureSets` is an optional table that can be used to link measures together. `sampleRelationships` is also optional and used only when you have samples that are pooled or split. `qualityReports` is used for detailed quality assurance and control measures.
+
+2\) **Contact information tables**. `sites`, `organizations`, `contacts`, and `addresses` hold contact information where testing was performed and who performed testing.
+
+3\) **Protocol or methods tables**. `protocols` and `instruments` are where you record information about the methods used to perform sample, or make measurements or observations. how samples are taken and measures are reported. `protocolSteps` and `protocolRelationships`tables are part used alongside the `protocols` table.
+
+4\) **Other report tables**. These tables include `polygons` where information is store about the sample shed and `datasets` which includes information about the data custodian and owner of the data.
+
+5\) **Dictionary tables**. `parts`, `sets`, `languages`, and `translations` these tables....
+
+<!--` reference1/ -->
+
+```{r, echo=FALSE, warning=FALSE}
+# Can put warning=FALSE above if warnings in display are an issue
+# Source utility functions
+source("../R/qmd-utils.R")
+
+parts_sheet_name <- "parts"
+path_to_excel <- "../data/raw/ODM-dictionary.xlsx"
+
+# Declaring string that will be appended during the loops
+table_of_content <- glue::glue('# Table of Contents
```

Comment:
This should be using a second level heading, ## since the Tables header at the very top is a single heading